lib/avro_turf/messaging.rb

```diff
@@ -89,6 +95,9 @@ def initialize(
           client_key_pass: client_key_pass,
           client_cert_data: client_cert_data,
           client_key_data: client_key_data,
+          oauth_url: oauth_url,
+          oauth_client_id: oauth_client_id,
+          oauth_client_secret: oauth_client_secret,
```

Comment:
lib/avro_turf/messaging.rb#130 shall we add a config param to enable/disable automatic registration of schemas?